MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpress.c

```diff
@@ -288,12 +308,23 @@ EnumerateNvmeDevNamespace (
                Device->DevicePath,
                &gEfiBlockIoProtocolGuid,
                &Device->BlockIo,
-               &gEfiBlockIo2ProtocolGuid,
-               &Device->BlockIo2,
                &gEfiDiskInfoProtocolGuid,
                &Device->DiskInfo,
                NULL
                );
+
+        // MU_CHANGE [BEGIN] - Request Number of Queues from Controller
+        if (Private->NumberOfDataQueuePairs > 1) {

```

Comment:
This check is sprinkled in the module a few times. Would it be better to make it a macro so that we can change it more easily.